cookbooks/fb_helpers/libraries/node_methods.rb

```diff
@@ -220,7 +220,7 @@ def fedora_family?
     end
 
     def macos?
-      %w{mac_os_x macos}.include?(self['platform'])
+      self['platform'] == 'mac_os_x'
```

Comment:
macos was never a platform in ohai

Reply:
I'm finding boxes which have node['platform'] as macos.  Can't get in to them so I can't offer any explanation.

Reply:
OK after looking through the history of Ohai and talking to Tim, I'm pretty sure this isn't true. I suspect that the Handlers you have are munging that, and it's not what `node['platform']` is giving you.

Nonetheless, I'll update this PR to keep `macos` for now, with a comment.

Reply:
Updated.